.env.example

```
@@ -33,22 +33,31 @@
-#   POSTGRES_HOST=<your-server-ip>
-POSTGRES_HOST=your-postgres-host
-POSTGRES_PORT=5432
-POSTGRES_DATABASE=your_database
-POSTGRES_USER=postgres
-# SECURITY: Replace this placeholder with a secure password
-# Generate using: openssl rand -base64 32
-POSTGRES_PASSWORD=__REPLACE_WITH_SECURE_PASSWORD__
+#   e.g., postgresql://role:pass@<your-server-ip>:5432/omnibase_infra
+OMNIBASE_INFRA_DB_URL=postgresql://role_omnibase_infra:__REPLACE_WITH_SECURE_PASSWORD__@your-postgres-host:5432/omnibase_infra
+OMNIINTELLIGENCE_DB_URL=postgresql://role_omniintelligence:__REPLACE_WITH_SECURE_PASSWORD__@your-postgres-host:5432/omniintelligence
+OMNICLAUDE_DB_URL=postgresql://role_omniclaude:__REPLACE_WITH_SECURE_PASSWORD__@your-postgres-host:5432/omniclaude
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The OMNICLAUDE_DB_URL key should go before the OMNIINTELLIGENCE_DB_URL key

(UnorderedKey)